requirements/cuda.txt

```diff
@@ -5,6 +5,7 @@ numba == 0.60.0; python_version == '3.9' # v0.61 doesn't support Python 3.9. Req
 numba == 0.61.2; python_version > '3.9'
 
 # Dependencies for NVIDIA GPUs
+pyarrow == 18.0.0 # temporary fix for missing pyarrow in ray until 2.44.2 ray release
```

Comment:
Can we update pyarrow to 20.0.0. The reason for that is 2.16 vllm image uses 20.0.0. IMO this doesn't make sense when older version has latest version and newer version has old version. I think we should maintain the consistency .

Reply:
I think latest pyarrow is 19.1 but when I use that, it says it clashes with some other requirement where pyarrow is set to 18.0. So I matched it to clear the error.

Reply:
Actually you are right, there is 20.0 released 3 days ago but I don't know if it's stable yet. Plus, the 2.16 image shows 19.0.0 version. But either way, upgrading to 19.1.0 gives an error saying some other dependency requires 18.0.0 pyarrow.

Reply:
Yeah you are right i overlooked the version in 2.16 .

But either way, upgrading to 19.1.0 gives an error saying some other dependency requires 18.0.0 pyarrow.

> Do you know which other packages require PyArrow 18.0.0? This seems unusual.

Reply:
It's in test.txt. let me try upgrading both and rebuild while I test the currently built image.